core/src/main/scala/ai/h2o/sparkling/backend/api/scalainterpreter/ScalaCodeResultV3.java

```diff
@@ -1,11 +1,12 @@
-package water.api.scalaInt;
+package ai.h2o.sparkling.backend.api.scalainterpreter;
 
 import water.Iced;
 import water.Key;
 import water.api.schemas3.KeyV3;
 
 public class ScalaCodeResultV3 extends KeyV3<Iced, ScalaCodeResultV3, ScalaCodeResult> {
-    public ScalaCodeResultV3() { }
+    public ScalaCodeResultV3() {
```

Comment:
nit: Could be on one line?

Reply:
👍  I have transformed this class to Scala ( Idea formats it on separate lines in case of Java)